dspace-api/src/main/java/org/dspace/curate/QueueManager.java

```diff
@@ -0,0 +1,102 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.curate;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.apache.log4j.Logger;
```

Comment:
This is Log4J v1.  Should be `org.apache.logging.log4j.Logger`.  Get one from `org.apache.logging.log4j.LogManager()`.

Reply:
Fixed in 352e6f1